pkg/apiserver/request/requestinfo.go

```diff
@@ -103,7 +103,7 @@ type RequestInfoResolver struct {
 // /api
 // /healthz
 // /
-func (r *RequestInfoResolver) GetRequestInfo(req *http.Request) (RequestInfo, error) {
+func (r *RequestInfoResolver) GetRequestInfo(req *http.Request) (*RequestInfo, error) {
```

Comment:
good question. happened in https://github.com/kubernetes/kubernetes/pull/33490/files#diff-07d66560df8f80f564e18b05364c82d1R106 and I didn't notice it.